src/main/java/io/github/protonmc/proton/module/decoration/TaterModule.java

```diff
@@ -17,7 +17,6 @@
 
     public TaterModule() {
         super(Proton.identifier("tater"));
-        this.enabled = false;
```

Comment:
This still shouldn't be enabled by default, Potato's are not the base of the mod.

Reply:
ok so how would you disable it

Reply:
Maybe we should ask the creator of tiny-config.

Reply:
Until then, I'll re-disable it for now.